src/mo_def/compUnit.ml

```diff
@@ -88,14 +89,15 @@ let obj_decs obj_sort at note id_opt fields =
 let decs_of_lib (cu : comp_unit) =
   let open Source in
   let { imports; body = cub; _ } = cu.it in
-  let import_decs = List.map (fun { it = (id, fp, ri); at; note}  ->
-    { it = LetD (
-      { it = VarP id; at; note; },
-      { it = ImportE (fp, ri);
-        at;
-        note = { note_typ = note; note_eff = Type.Triv} });
-      at;
-      note = { note_typ = note; note_eff = Type.Triv } }) imports
+  let import_decs =
+    List.map (fun { it = (pat, fp, ri); at; note} ->
+        { it = LetD (pat,
+                     { it = ImportE (fp, ri);
+                       at;
+                       note = { note_typ = note; note_eff = Type.Triv} });
+          at;
+          note = { note_typ = note; note_eff = Type.Triv } }
+      ) imports
```

Comment:
Use the identation style on the left - I think that's what Andreas was advocating... (at least my understanding of it).

Reply:
This is what `emacs` gave me, will change.

Reply:
b5ed7d031ceb01527c84dfe0107c596e815616a8